test/unit/argument_parser/detail/format_html_test.cpp

```diff
@@ -48,9 +48,12 @@ TEST(html_format, empty_information)
                            "<dd>Whether to to check for the newest app version. Default: 1.</dd>\n"
                            "</dl>\n"
                            "<h2>Version</h2>\n"
-                           "<strong>Last update:</strong> <br>\n"
-                           "<strong>empty_options version:</strong> <br>\n"
-                           "<strong>SeqAn version:</strong> " + seqan3::seqan3_version + "<br>\n"
+                           "<p>\n"
+                           "<strong>Last update: </strong>\n"
+                           "<br>\n"
+                           "<strong>empty_options version: </strong>\n"
+                           "<br>\n"
+                           "<strong>SeqAn version: </strong>" + seqan3::seqan3_version + "\n"
```

Comment:
I'm not an html expert, but aren't we missing a `</p>` tag here?

Reply:
Good catch!

Reply:
This was even a fix to the previous version without the refactoring!

Reply:
Great! :smile: